news/591.minor-changes

```diff
@@ -0,0 +1 @@
+Keep GitHub Actions up to date with GitHub's Dependabot
```

Comment:
Hm. Usually, we record these changes in this file:
 https://github.com/collective/icalendar/blob/master/CHANGES.rst

Would you be willing to delete the file and mention the content there, so it is recorded in the changelog?

Thanks again for the PR!